src/index.js

```diff
@@ -676,6 +698,24 @@ function dataLoaded(geomapFeatures, allDates, baseData) {
     const slider = resetSlider(allDates);
     resetGeoMap(geomapFeatures);
 
+    const top5 = svg.append("g")
+        .attr("class", "top5");
+
+    var xValTop5 = 40;
+    var yValTop5 = 360;
+    top5.append("text").attr("class", "top5Text").attr("x", xValTop5).attr("y", yValTop5 - 20)
+    for (let i = 0; i < 5; i++) {
```

Comment:
[nit] You shouldn't ever have to change D3 elements in a loop. The data() function lets you put an array of data into a an element and it automatically creates elements for you when you use enter(), and you can automatically update the elements when you change the data. (Google D3 data() and enter() for more info)

In this example, you could start with an array of values you want to put in the top5.
const top5Values = [{name: name, value: value}, ...];
Then enter it into data() to create new elements
svg.selectAll(".top5Text")
    .data(top5Values)
    .enter().append("text")
        .attr("class", "top5Text")
        .attr("x", xValTop5)
        .attr("y", (d, i) => yValTop5 + (i * 20));
Since this is a hackathon, it might be too much effort to fix, but this data() binding is super powerful and is sort of the whole point of D3.